releasenotes/notes/Added-new-methods-added-to-BackendProperties-71e60b610cc4bcc7.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    Added 5 new commands to easily retrieve user-specific data from
+    ``BackendProperties``: ``gate_property``, ``gate_error``, ``gate_length``,
+    ``qubit_property``, ``t1`` and ``frequency``. They return the specific values
+    of backend properties. For example::
+
+      from qiskit.test.mock import FakeOpenPulse2Q
```

Comment:
Maybe the example can be done with a real device?

Reply:
👍 I didn't know it was okay to use a client device in the example; that would be nice, although it wouldn't change the method demonstration. We could also maybe replace lines 9 and 10 with something like `# `backend` is a pulse enabled `BaseBackend` instance`